rasa/nlu/training_data/message.py

```diff
@@ -117,13 +120,16 @@ def get_combined_intent_response_key(self) -> Text:
         return f"{intent}{response_key_suffix}"
 
     @staticmethod
-    def separate_intent_response_key(original_intent) -> Optional[Tuple[Any, Any]]:
+    def separate_intent_response_key(
+        original_intent: Text,
+    ) -> Tuple[Optional[Text], Optional[Text]]:
 
         split_title = original_intent.split(RESPONSE_IDENTIFIER_DELIMITER)
         if len(split_title) == 2:
             return split_title[0], split_title[1]
         elif len(split_title) == 1:
             return split_title[0], None
+        return None, None
```

Comment:
We should actually raise an exception here. This will only happen if the intent name has more than one `/`(`RESPONSE_IDENTIFIER_DELIMITER`) which is not a valid intent name.

Reply:
happy to raise here. What type of exception should I raise?

Reply:
Maybe a ValueError, right? bc the value of `original_intent` is invalid. In the description we can explain the above mentioned reason.

Reply:
ah yeah ok, I thought we had custom exception classes. Will use the right standard one then

Reply:
Ohh right, `RasaException` in `rasa/exceptions.py`. None of the other custom exception descriptions fit the case here.